Dockerfile

```diff
@@ -29,6 +31,18 @@ COPY api/ api/
 COPY controllers/ controllers/
 COPY pkg/ pkg/
 
+# Add kubectl tool
+RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/${TARGETARCH}/kubectl"
+RUN chmod +x ./kubectl
+
+# Add helm chart DIR to get CRDs from it
+COPY deployment/network-operator chart
+# Update helm dependencies and copy CRDs from them
+RUN mkdir crds && \
+    cd ./chart && \
+    cp -r ./crds /workspace/crds/network-operator/ && \
+    cp -r charts/sriov-network-operator/crds /workspace/crds/sriov-network-operator/
```

Comment:
do we want NFD CRDs ? or we going to have issues with NFD deployed from GPU Op ?